src/main/java/com/vinyl/service/UserService.java

```diff
@@ -36,6 +36,7 @@ public User addUser(User user) {
 		user.setRole(r);
 
 		accountValidator.validateNames(user);
+		accountValidator.validateEmail(user);
```

Comment:
the task specified to create a separate class that handles email validation, please move the email validation to it's own class

Reply:
done